app/models/manageiq/providers/microsoft/infra_manager/template.rb

```diff
@@ -7,6 +7,8 @@ class ManageIQ::Providers::Microsoft::InfraManager::Template < ManageIQ::Provide
     end
   end
 
+  supports_not :clone, :reason => 'Cloning templates is not supported for SCVMM'
```

Comment:
You want to wrap the translation in `_()` for i18n

But then, I'd just discard the reason and let the mixin display it's generic reason